app/commands/ingest.py

```
@@ -100,95 +100,99 @@ def run():
 
 
-@channels.command(name="list")
-def list_channels():
-    """List all monitored channels."""
+@sources.command(name="list")
+def list_sources():
+    """List all monitored sources."""
     url = get_transcription_url()
     try:
-        response = requests.get(f"{url}/ingestion/channels")
+        response = requests.get(f"{url}/ingestion/sources")
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)